e2e/rbd_helper.go

```diff
@@ -72,6 +72,7 @@ func createRBDStorageClass(c kubernetes.Interface, f *framework.Framework, scOpt
 		mOpt := strings.Split(opt, ",")
 		sc.MountOptions = append(sc.MountOptions, mOpt...)
 	}
+	sc.ReclaimPolicy = &retainPolicy
```

Comment:
this is also a global variable in `e2e/utils.go`, use `policy` here to refer to the parameter of the function